src/RulesTest/Library/BoundingRectangleCompletelyObscuresContainerTest.cs

```diff
@@ -165,5 +166,31 @@ public void BoundingRectangleCompletelyObscuresContainer_BottomFail()
             Assert.IsTrue(Rule.Condition.Matches(e));
             Assert.AreEqual(EvaluationCode.Error, Rule.Evaluate(e));
         }
+
+        [TestMethod]
+        public void BoundingRectangleCompletelyObscuresContainer_IsDialog_NotApplicable()
```

Comment:
Should there also be a test that ensures that a Window is excluded?

Reply:
Added.